src/controllers/controller.bar.js

```diff
@@ -290,17 +290,17 @@ export default class BarController extends DatasetController {
 
 			// all borders are drawn for floating bar
 			/* TODO: float bars border skipping magic
-			if (me._getParsed(i)._custom) {
+			if (me.getParsed(i)._custom) {
```

Comment:
Can we just remove this commented code?

Reply:
Maybe let's handle separately? It may be helpful to leave this until we fix the TODO and fixing the TODO seems big enough that it should be a separate item. I put a reminder in the v3 tracking bug to go through and fix any TODOs so that we don't forget